.atomist/editors/RugEditorsPredicates.ts

```diff
@@ -19,19 +19,31 @@ import { Project } from "@atomist/rug/model/Project";
 import deprecated from "deprecated-decorator";
 
 export function isRugArchive(p: Project): boolean {
-    return p.fileExists(".atomist/manifest.yml");
-}
-
-export function isSetUpForTypeScript(p: Project): boolean {
+    if (p.fileExists(".atomist/manifest.yml")) {
+        p.editWith("ConvertManifestToPackageJson", {});
```

Comment:
this is a side-effecting getter.

I can't like I can't not remark on that evil
even though I think it is a good thing to accomplish

Can we console.log to let people know why we're doing that?